### DISCUSSION TITLE
Cannot Install

### DISCUSSION THREAD

**Discussion by HopWorks** [bluenviron/mediamtx#2494] [Questions] [answered]
### Question

I hope I am not missing anything obvious, but I cannot install this on my Raspberry Pi 4 Model B running Bookworm, Bullseye, Buster, DietPi, etc. Of the 4 options given to install, a binary is out or at least I never installed a binary on a distro, Docker is out because I am not running docker, Arch Linux does not work because even though I installed pacman, "MAKEPKG" is command not found.

Is there a DEB of this offering? 

I apologize if my newbie ignorance is missing the obvious. I have not installed a lot of Git Clones but still a number of them and never encountered where I had to build using the Arch Linux package manager before.

And many of the solutions I researched to RTSP stream a Raspberry Pi IMX296 camera points to this software.

Thanks

**Comment by Ernovation** [comment, marked as answer]
The binary images contain just the executable file (called mediamtx) and a configuration file (mediamtx.yml). You do not need to install anything, just run the executable file from the shell, with the configuration file in your current directory.

**Comment by aler9** [comment]
I'm considering producing Debian packages in order to solve such problems, feature request is here: #1794

[a comment by an automated account is not shown]
